src/subdomains/generic/kyc/services/kyc.service.ts

```diff
@@ -511,12 +535,24 @@ export class KycService {
       case KycStepName.PERSONAL_DATA:
         return { nextStep: { name: nextStep, preventDirectEvaluation }, nextLevel: KycLevel.LEVEL_10 };
 
+      case KycStepName.LEGAL_ENTITY:
+        return { nextStep: { name: nextStep, preventDirectEvaluation } };
+
+      case KycStepName.STOCK_REGISTER:
+        return { nextStep: { name: nextStep, preventDirectEvaluation } };
+
       case KycStepName.NATIONALITY_DATA:
         return { nextStep: { name: nextStep, preventDirectEvaluation }, nextLevel: KycLevel.LEVEL_20 };
```

Comment:
Looks like we need to improve the `nextLevel` thing 🤔  Level 20 should be set when completing personal data.